.github/scripts/inductor_accuracy_summary.py

```diff
@@ -0,0 +1,369 @@
+## python inductor_accuracy_summary.py -t TRITON_v3.0 -r TRITON_v2.1 -p amp_bf16 amp_fp16 bfloat16 float16 float32 -s torchbench

```

Comment:
Remove Triton in the example cmd. Can we merge the accuracy summary script with performance scripts as one script?

Reply:
Removed Triton in cmd. The data format in accuracy and performance is different and it need more effort. It can be enhancement in the future.